.changeset/tidy-radios-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/highcharts-theme": patch
+---
+
+Add Salt Highcharts theme
```

Comment:
can we we expand on this, which version does this target, no mention of value being provided to end-user, in this case LineChart, so folks may think this is everything.